scripts/replacePackageVersion.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * This script will accept a path to a package.json file, dependency name, and a version string,
+ * and replace the version of the dependency in the package.json file.
+ *
+ * This script will attempt to replace both "dependencies" and "peerDependencies" in the package.json file,
+ * and it will output success if it finds the dependency in either of them.
+ * It will output an error if the dependency is not found in either of them.
+ *
+ * If you want to pass a filename instead of a version, then that filename should be
+ * prefixed with `file:`.
+ *
+ * Usage: node scripts/replacePackageVersion.js <package.json-path> <dependency-name> <version>
+ *
+ * Example with a version:
+ * node scripts/replacePackageVersion.js ./package.json react 18.0.0
+ *
+ * Example with a file:
+ * node scripts/replacePackageVersion.js ./package.json react file:./react-18.0.0.tgz
+ */
+
+const fs = require('fs');
+const {TestResult} = require("./TestResult.js");
+
+exports.replacePackageVersion = function(packageJsonPath, dependencyName, version) {
```

Comment:
Why not write with es6 syntax?

Reply:
I didn't want to deal with the mjs setup and filenames and whatnot.